### PR TITLE
11.0 6f3ec0a690fe100ed417fe0da020eca1527dd8fe

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: pos_mobile_restaurant/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "category": "Point of Sale",
     "live_test_url": "http://apps.it-projects.info/shop/product/pos-mobile-ui?version=11.0",
     "images": ["images/pos_mobile_restaurant.png"],
-    "version": "11.0.1.3.8",
+    "version": "10.0.1.3.8",
     "application": False,
```
Comment (Ramil-Mukhametzyanov) [inline pos_mobile_restaurant/__manifest__.py]: Incorrect change

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
I paid for Ekomurz.nl (Pos Category: Partner Services) using Credit (Fruits & Vegetables only)
fruit credit

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
The last error has been fixed.

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
multi_session does not work

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
pos_multi_session and pos_printer_network become uncompatible.
Set printer to Admin's POS and enter. Then after enter to Demo's POS, an error occurs.

urlindexof

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
@yelizariev @ilmir-k 
please check it

**Review by ilmir-k — APPROVED** [review]

**Pull request merged**